src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/IFaceOperations.cs

```diff
@@ -144,10 +144,7 @@ public partial interface IFaceOperations
         /// <exception cref="Microsoft.Rest.SerializationException">
         /// Thrown when unable to deserialize the response
         /// </exception>
-        /// <exception cref="Microsoft.Rest.ValidationException">
-        /// Thrown when a required parameter is null
-        /// </exception>
-        Task<HttpOperationResponse<VerifyResult>> VerifyWithHttpMessagesAsync(string faceId1, string faceId2, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<HttpOperationResponse<VerifyResult>> VerifyWithHttpMessagesAsync(System.Guid faceId1, System.Guid faceId2, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
s/VerifyWithHttpMessagesAsync/VerifyFaceToFaceWithHttpMessagesAsync/g

Just make it specific to avoid potential misleading.

Reply:
Done